Update issue templates

amoerie opened the pull request:
Hey @gofal , I found out about this Github feature via other open source projects, and it's quite nice: when people create a new issue, they can choose between any of the existing templates, or make an empty one. The benefit is that it "guides" the user into filling in the correct details, and it requires a little less typing of people.

Let me know what you think, I think it's neat. :-)

gofal commented:
Uh, nice feature!
I observed three types of issues so far: bugs, feature requests, and questions (some kind of "how can I do that with fo-dicom?").
up to now I used the labels "Bug", "Question" and "Enhancement" to classify the issues. If the issues are created and are automatically assigned with the labels, then we should think about a way how to mark the issues as classified. What I mean is that often users claim a bug, but in fact they want do something that is not yet implemented and it is acutally an enhancement. Or they missed something and it really was only a question.
I'm using the "bug" label very intensively because for me it is a trigger to see when a next release is fine (when there are no bugs open, that could be fixed quickly).

amoerie commented:
I get what you're saying. So perhaps we should add a label to each new issue with "evaluating" or "new" or "to review"?

amoerie commented:
@gofal 

- I've updated the bug report template to use the labels "bug" and "new"
- I've updated the feature request template to use the labels "enhancement" and "new"

The "new" marks it as a 'new' issue that needs to be reviewed by one of us. Is that okay? The alternative option would be to only have a "new" label, and we would add the "bug" or "enhancement" label when we review it. What is your preference :-)

gofal commented:
Thanks. My preference would be to have just a "new" label or a "new-bug" or "new-enhancement" that will be changed into an actual "bug", "enhancement", "question", "wontfix" or whatever. But I am also fine with having both labels as it is now.

gofal commented on .github/ISSUE_TEMPLATE/bug_report.md:
```diff
@@ -0,0 +1,20 @@
+A clear and concise description of what you expected to happen.
+
+**Screenshots or test DICOM files**
+If applicable, add screenshots or test DICOM files to help explain your problem.
```
When it comes to bugs, it is always important to describe the environement, like which version of fo-dicom, and on which platform (windows, linux, 32bit, 64bit, ..)

gofal replied:
thats included in the current issue-template.

amoerie replied:
I just saw, nice catch! I've updated the template.

amoerie commented:
No you're right, I like just the 'new' label too. Changed!
